doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -401,6 +414,31 @@ Displaying information about notations
    If on, parentheses are printed even if implied by associativity and precedence
    Default is off.
 
+
+.. cmd:: Print Grammar @ident
+
+   Shows the grammar for the nonterminal :token:`ident`, which must be one of the following:
+
+   - `constr` - for :token:`term`\s
+   - `pattern` - for :token:`pattern`\s
+   - `tactic` - for currently-defined tactic notations, :token:`tactic`\s and tacticals with
+   - `vernac` - for :token:`command`\s
+
+   The first three of these give the precedence and associativity for each construct.
+   For example, these lines printed by `Print Grammar tactic` indicates that the `try` construct
+   is at level 3 and right-associative.  `SELF` represents the `tactic_expr` nonterminal
+   at level 5 (the top level)::
+
+     | "3" RIGHTA
+       [ IDENT "try"; SELF
+
+   Note that the productions printed by this command are represented in the form used by the
+   XXX parser, which differs from how productions are shown in the documentation.  The grammar
+   described in this documentation is equivalent to the grammar of the |Coq| parser, but has been
+   heavily edited to improve readability and presentation.
+
+   .. todo PR  XXX above = ??  Can we link to a description?
```

Comment:
You mean the "coqpp parser"? Well, no, there is no description available. Just say "the internal coqpp parser".

Reply:
It's not quite internal, since it needs to be used by plugin developers writing .mlg files